ex-03/readme.md

```diff
@@ -83,13 +89,89 @@ Snyk has a SAST (Static Application Security Testing) module, let's try it.
   trufflehog git file://../.. --no-update
   ```
 
+- Explore Trufflehog options
+
+  ```shell
+  trufflehog --help
+  ```
+
 ### -- Discussions --
 
 - The first scan (the filesystem) found the secret in the .txt file, but not in the app.py?
 - The second scan (git) scanned the whole git commit history
 
-## Take away
+## Secret verification
+
+Many tools have will try to *verify* if a secret is valid. The verification part involves using the secret against the service which the scanner believes it is for. In Trufflehog this is default if the detectors that supports it. For the example of the GitHub token, Trufflehog's [detector](https://github.com/trufflesecurity/trufflehog/blob/0d8c3335ed9321c2e198206f9279a31c8f2e3b67/pkg/detectors/github/v1/github_old.go#L109) will hit the github api using the secret. If the api call is successfull Trufflehog will report "Verified" - else "Found unverified result 🐷🔑❓"
```

Comment:
Should we instead point to the "new" github-scanner?
https://github.com/trufflesecurity/trufflehog/blob/0d8c3335ed9321c2e198206f9279a31c8f2e3b67/pkg/detectors/github/v2/github.go#L49

Reply:
The current link point to the "verification routine", we'll keep this.